src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('.tree');
+const elements = document.querySelectorAll('li');
+
+for (const element of elements) {
+  if (element.firstElementChild) {
+    const span = document.createElement('span');
+
+    span.textContent = element.firstChild.textContent;
+
+    element.firstChild.replaceWith(span);
+  }
```

Comment:
Так було би читабельніше, як на мене)
```suggestion
  if (!element.firstElementChild) {
    continue;
  }
  
  const span = document.createElement('span');

  span.textContent = element.firstChild.textContent;
  element.firstChild.replaceWith(span);
  }
```